cmake/templates/4CConfig.cmake.in

```diff
@@ -0,0 +1,122 @@
+@PACKAGE_INIT@
+
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/modules ${CMAKE_MODULE_PATH})
+
+include(CMakeFindDependencyMacro)
+include(FetchContent)
+
+include("${CMAKE_CURRENT_LIST_DIR}/4CSettings.cmake")
+
+if(4C_WITH_MPI)
+  find_package(MPI REQUIRED)
```

Comment:
This is tricky, since this code is called from the call site, not from 4C. So it needs to get the dependencies. Exporting 4C alone does not provide this information.